reference/dic_tags.rst

```diff
@@ -370,6 +370,17 @@ the :class:`Symfony\\Component\\HttpKernel\\CacheWarmer\\CacheWarmerInterface` i
         public function warmUp($cacheDirectory)
         {
             // ... do some sort of operations to "warm" your cache
+
+            // this method must return an array with the file paths of the
```

Comment:
With file paths (without the?)
I am not an english native, but reading the file paths seems weird